tests/Unit/Domain/FunctionsOfTime/Test_Tags.cpp

```diff
@@ -18,17 +18,20 @@ struct Metavariables {
 };
 
 SPECTRE_TEST_CASE("Unit.Domain.FunctionsOfTime.Tags", "[Domain][Unit]") {
-  TestHelpers::db::test_simple_tag<Tags::FunctionsOfTime>("FunctionsOfTime");
+  TestHelpers::db::test_base_tag<Tags::FunctionsOfTime>("FunctionsOfTime");
+  TestHelpers::db::test_simple_tag<Tags::FunctionsOfTimeInitialize>(
+      "FunctionsOfTimeInitialize");
```

Comment:
I thought the name should be that of the simple tag here

Reply:
I'm confused, I don't see an issue. "FunctionsOfTimeInitialize" *is* the name of the simple tag now. "FunctionsOfTime" is the name of the base tag now.

Reply:
Oh sorry, what I meant is that I thought both the simple and the base tag would have the same name, presumably that of the base tag. That's how compute tags work. But don't worry about this, it's nothing to be concerned about in this PR.

Reply:
Compute tags take the name of the simple tags, simple tags tag their own name. I think these simple tags should have a name function that returns `FunctionsOfTime` without the `Initialize` unless the tags are going to be removed after initialization.

Reply:
No the tags aren't removed after initialization. I added a static name function that returns `FunctionsOfTime` to the simple tag and changed the test accordingly (squashed immediately)

Reply:
Thanks!